build.gradle

```diff
@@ -0,0 +1,89 @@
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'scala'
+
+    apply from: "${rootProject.projectDir}/gradle-scalatest.plugin"
+
+    ext {
+        ////////////////////////////////
+        // Commonly used dependencies //
+        ////////////////////////////////
+
+        mkDepGen = { genOne -> { String... names -> names.collect(genOne) } }
```

Comment:
I have projects that put nothing into gradle.properties, so not sure if
that's best. Rewriting is only important when you need to force a version,
so I agree it isn't natural for many things.